Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -39,6 +39,12 @@ struct BorderData {
     float width { 0 };
 };
 
+struct FlexBasisData {
+public:
```

Comment:
No need for `public:` since this is already a `struct` which is `public` by default. :)

Reply:
I just copied what `struct BorderData` did a few lines above. ;)
I think I got this comment from you before... Apparently I don't learn.